frontend/packages/pipelines-plugin/src/components/repository/RepositoryDetails.tsx

```diff
@@ -19,18 +30,40 @@ const RepositoryDetails: React.FC<RepositoryDetailsProps> = ({ obj: repository }
         <div className="col-sm-6">
           <ResourceSummary resource={repository} />
         </div>
-        {spec?.url && (
-          <div className="col-sm-6" data-test="pl-repository-customdetails">
+        <div className="col-sm-6" data-test="pl-repository-customdetails">
+          {spec?.url && (
             <dl>
               <dt>{t('pipelines-plugin~Repository')}</dt>
-              <dd>
+              <dd className="details-item">
                 <ExternalLink href={spec?.url}>
                   {getGitProviderIcon(spec?.url)} {spec?.url}
                 </ExternalLink>
               </dd>
             </dl>
-          </div>
-        )}
+          )}
+          {spec?.git_provider?.webhook_secret?.name && (
+            <dl>
+              <dt>{t('pipelines-plugin~Webhook Secret')}</dt>
+              <dd className="details-item">
+                <ResourceLink
+                  kind={SecretModel.kind}
+                  name={spec?.git_provider?.webhook_secret?.name}
+                  namespace={repository.metadata.namespace}
+                />
+              </dd>
```

Comment:
Sometimes it's easier to render the dt, dd, and its not completely wrong. On many pipelines details pages, we do this as well because the data is extracted from labels.

But here we should use the mentioned `DetailsItem` because it has a great feature, it allows the user do click on the label to get where the data comes from:

![image](https://user-images.githubusercontent.com/139310/229156837-142eaeac-2e6e-46fc-b2c7-04f831dc29ca.png)

That's a small but useful feature for the user.

```suggestion
              <DetailsItem
                label={t('pipelines-plugin~Webhook Secret')}
                obj={repository}
                path="spec.git_provider.webhook_secret.name"
              >
                <ResourceLink
                  kind={SecretModel.kind}
                  name={spec?.git_provider?.webhook_secret?.name}
                  namespace={repository.metadata.namespace}
                />
              </DetailsItem>
```

 In my memory it shows also the description from the CRD. Not sure why this doesn't work in this case.  But the description is also not super useful here. But its great that we don't need to maintain this ourself. If the description changed in the future, our UI will show the latest information.

You can find the schema when you search for the tekton repositories CRD or you can find it here:

http://localhost:9000/k8s/cluster/customresourcedefinitions/repositories.pipelinesascode.tekton.dev/yaml

![image](https://user-images.githubusercontent.com/139310/229157410-a20c794c-e736-4b17-8120-8d101ba0a3b9.png)